src/patternfly/components/Table/table.scss

```diff
@@ -144,6 +148,9 @@
   --pf-c-table__sort__button--active__sort-indicator--Color: var(--pf-global--Color--100);
   --pf-c-table__sort__button--focus__sort-indicator--Color: var(--pf-global--Color--100);
 
+  // Help
+  --pf-c-table__help--MinWidth: 11ch;
```

Comment:
wdyt about just using a modifier with a base min-width of 11, then modifying `.pf-c-table__sort.pf-m-help` to 15?